### PR TITLE
Added page

### PR DESCRIPTION
**Pull request by sn95**

### CODE REVIEW

**Comment by sn95** [comment]
@githubteacher 
Hi, I really don't understand why this isn't working. I looked at some other people's and the date format I've used looks the same as theirs

**Comment by Justme764** [comment]
Hello sn95,  I had the same issue.  remove the 3 dashes at the top and bottom and make it 3 dots.   Not sure about the time stamp didn't use one.  Hope this helps.

File: _pages/sn95.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: sn95
+pageUrl: sn95.github.io
+timestamp: 2017-12-13
+---
```
Comment (hectorsector) [inline _pages/sn95.yaml]: @sn95 you just need a line break after line 5!

**Comment by crichID** [comment]
@sn95 I went ahead and added a line break in https://github.com/githubschool/on-demand-github-pages/pull/1823/commits/4e6a066677bb666217e2d7ea4fa64af35b30ccab. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by December 29th, I will go ahead and do it just to keep the repository clean!

File: _pages/sn95.yaml (hunk 1 above)
Comment (sn95) [reply _pages/sn95.yaml]: Thank you!

**Pull request merged**